SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -70,7 +73,7 @@ Buffer {
 
 	allocMsg { arg completionMessage;
 		this.cache;
-		^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, completionMessage.value(this), sampleRate !? _.asFloat ?? 0.0]
+		^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, completionMessage.value(this), sampleRate]
```

Comment:
we want to keep `sampleRate !? _.asFloat ?? 0.0` here, since allocMsg can be used directly, and nil.asFloat wouldn't work

Reply:
I think this one slipped your last pushes. It should be:
```supercollider
^["/b_alloc", bufnum, numFrames.asInteger, numChannels.asInteger, completionMessage.value(this), sampleRate !? _.asFloat ?? 0.0]
```
we need that check because allocMsg can be used on its own, and then we need to make sure sampleRate is valid and float